src/postgres/commands/PostgresQueryWizard/FunctionQuerySteps/FunctionQueryNameStep.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from "vscode-azureextensionui";
+import { ext } from "../../../../extensionVariables";
+import { localize } from "../../../../utils/localize";
+import { PostgresServerTreeItem } from "../../../tree/PostgresServerTreeItem";
+import { IPostgresFunctionQueryWizardContext } from "./IPostgresFunctionQueryWizardContext";
+
+export class FunctionQueryNameStep extends AzureWizardPromptStep<IPostgresFunctionQueryWizardContext> {
+    public async prompt(wizardContext: IPostgresFunctionQueryWizardContext): Promise<void> {
+        wizardContext.name = (await ext.ui.showInputBox({
+            prompt: localize('provideFunctionName', 'Provide function name'),
+            validateInput: PostgresServerTreeItem.validateIdentifier
+        })).trim();
+    }
```

Comment:
Can we move `validateIdentifier` to this file? I only see it used here, so I'm not entirely sure why it's in `PostgresServerTreeItem.ts`